chef/cookbooks/glance/templates/default/glance-api.conf.erb

```diff
@@ -81,6 +81,7 @@ workers = <%= [node["cpu"]["total"], 2, 4].sort[1] %>
 
 # Supported values for the 'container_format' image attribute
 #container_formats=ami,ari,aki,bare,ovf,ova
+container_formats=ami,ari,aki,bare,ovf,ova<%= @use_docker ? ",docker" : "" %>
```

Comment:
For the record, the reason to not use an attribute array is that we want to easily be able to compare the values we don't change (`ami,ari,aki,bare,ovf,ova`) with what is in the upstream config file.